src/typescript/index.ts

```diff
@@ -25,6 +25,7 @@ async function main() {
       try {
         const openapiDefinitionDocument = await initiator.ReadFile(file)
         const openapiDefinitionObject = safeLoad(openapiDefinitionDocument)
+
```

Comment:
This file can be ignored since adding blank line only.

Reply:
reverted